### PR TITLE
dpll_mon and rebuild priority fixes

### PR DESCRIPTION
**Pull request by kubalewski**
Fixes shall address the issues with priority rebuild and initialization of dpll_mon.

### CODE REVIEW

**Comment by mlichvar** [comment]
I repeated my tests from #37 and it all seems to work nicely now. It's deterministic and doesn't connect one port to both clocks. The code looks good to me. Just noticed few typos in commit messages if you care about that.

Thanks!

File: synce_dev.c [hunk 1]
```diff
@@ -617,15 +618,21 @@ int rebuild_inputs_prio(struct synce_dev *dev)
...
 				break;
 		if (i == prio_count)
 			synce_clock_source_prio_clear(dev->dpll_mon, tmp);
 	}
-	for (i = 0; i < prio_count; i++)
+	/* update priorities on valid sources */
+	for (i = 0; i < prio_count; i++) {
+		if (!synce_clock_source_prio_get(dev->dpll_mon, arr[i], &prio))
+			if ((int)prio == i)
+				continue;
```
[the remark this answers is not in the document]
Comment (kubalewski) [reply synce_dev.c:633]: sure, we don't want to resend/change priority of the source if on previous rebuild the same priority value for this source was assigned

[the remark this answers is not in the document]
Comment (mwolech) [reply synce_dev.c:633]: But the prio is still assigned if the synce_ext_src_prio_get returns a non-zero value ?

[the remark this answers is not in the document]
Comment (kubalewski) [reply synce_dev.c:633]: Yes, that's error condition, it failed to obtain "current" priority of the source, thus we just send new one.

[the remark this answers is not in the document]
Comment (mwolech) [reply synce_dev.c:633]: Ah, I see. Thanks!

**Review by mwolech — APPROVED** [review]

**Comment by kubalewski** [comment]
updated a commit messages to get rid of typos

**Pull request merged**